### DISCUSSION TITLE
Migrating from Pootle

### DISCUSSION THREAD

**Discussion by xperseguers** [WeblateOrg/weblate#7029] [Q&A] [answered]
Hi there,

I'm a complete newbie with Weblate but used to administrate Pootle for quite some years now.

I try to get my head around the concepts of Weblate but for various reasons I don't want to integrate with git at this point and would like to keep dedicated "local repo" within Weblate with my various projects.

In Pootle you basically have various separate "projects". In Weblate afaiu you have a "Project" and then "Components" which are logically the Pootle projects I had with a way to group them together when they relate to various components of a same bigger project. So far so good.

I've various projects in Pootle but +/- all of them are related to TYPO3 extensions. TYPO3 extension localization files are stored this way:

```
<extension-name>
|-- ...
|-- Resources
|   `-- Private
|       `-- Language
|           |-- locallang.xlf
|           |-- locallang_db.xlf
|           |-- ...
...
```

So I need to be able to translate multiple XLF files within a so-called "component" in Weblate.

I don't care about the directory structure so I prepared a zip with existing (let say French and German) translations this way:

```
.
|-- en
|   |-- locallang.xlf
|   `-- locallang_db.xlf
|-- de
|   |-- locallang.xlf
|   `-- locallang_db.xlf
`-- fr
    |-- locallang.xlf
    `-- locallang_db.xlf
```

The English file is of the form:

```
<?xml version="1.0" encoding="utf-8" standalone="yes" ?>
<xliff version="1.0">
	<file source-language="en" datatype="plaintext" original="messages" date="2015-05-28T07:56:41Z" product-name="file_list">
		<header/>
		<body>
			<trans-unit id="filename">
				<source>Filename</source>
			</trans-unit>
			<trans-unit id="info">
				<source>Info</source>
			</trans-unit>
```

while a translation is of the form:

```
<?xml version='1.0' encoding='utf-8'?>
<xliff version="1.0">
	<file source-language="en" datatype="plaintext" original="messages" date="2015-05-28T07:56:41Z" product-name="file_list" target-language="fr">
		<header/>
		<body>
			<trans-unit id="filename" approved="yes">
				<source>Filename</source>
				<target state="translated">Nom du fichier</target>
			</trans-unit>
			<trans-unit id="info" approved="yes">
				<source>Info</source>
				<target state="translated">Info</target>
			</trans-unit>
```

Problem pops up when I try to create such component by uploading a zip archive... Somehow it looks like only 1 single XLIFF may be configured. But in that case I have two of them.

Or did I miss something?

**Comment by xperseguers** [comment, marked as answer]
Looking at another question https://github.com/WeblateOrg/weblate/discussions/7001, I get the feeling that in my example, `locallang.xlf` would be one "component" `locallang` and the second file `locallang_db.xlf` would be another component e.g., `locallang_db`. Is that correct? in that case, components are indeed "files" and not Pootle projects having multiple files. This means I actually cannot group together various (TYPO3) extensions into a logical bigger "project", is that correct?

**Reply by nijel** [reply]
Not right now, but it will be possible in the future, see https://github.com/WeblateOrg/weblate/issues/263
